README.md

```diff
@@ -73,12 +73,15 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 60 | [Music Token Metadata](./CIP-0060) | Active |
 | 67 | [Asset Name Label Registry](./CIP-0067) | Proposed |
 | 68 | [Datum Metadata Standard](./CIP-0068) | Proposed |
+| 71 | [Non-Fungible Token (NFT) Proxy Voting Standard](./CIP-0071) | Proposed |
 | 74 | [Set min-pool-cost to 0](./CIP-0074) | Proposed |
+| 83 | [Encrypted Transaction message/comment metadata (Addendum to CIP-0020)](./CIP-0083) | Active |
```

Comment:
I think we have to wait for the link https://github.com/cardano-foundation/CIPs/blob/editors-meeting-60-housekeeping/CIP-0083 to be valid (i.e. merging https://github.com/cardano-foundation/CIPs/pull/409) before merging this (asked if we're ready in https://github.com/cardano-foundation/CIPs/pull/409#issuecomment-1385337153).